library/src/morpheus-master/sass/base/_responsive.scss

```diff
@@ -1,7 +1,14 @@
 @import "base/compass";
 
 body{
-	@media #{$phone}{
+	@media #{$phone}
+	{
```

Comment:
This file does not use Allman style braces. Can we keep it consistent?

Reply:
Yeah, I agree. It had K&R style (which I personally like over Allman). I think Allman and K&R are used inconsistently throughout Sakai, but nice to at least keep the file to the same style.

Reply:
Yeah everyone has their favourite. Personally I prefer Allman, but my rule of thumb when working in Sakai is to 'keep it consistent': I try my best to use whatever styles are already most prevalent in the given file.

It's the mixing of styles and whitespace that makes reading some Sakai code a nightmare...